tests/plotting/common_test.py

```diff
@@ -70,3 +70,19 @@ def test_preprocess_raises_for_unsupported_dtype(dtype_and_shape):
         match=f'The input has dtype {dtype} which is not supported by Plopp',
     ):
         preprocess(v)
+
+
+def test_preprocess_drops_coords_that_are_not_plotted_dim_coords():
+    da = data_array(ndim=2)
+    da.coords['xx2'] = 7.5 * da.coords['xx']
+    da.coords['yy2'] = 3.3 * da.coords['yy']
+    out = preprocess(da)
+    assert set(out.coords) == {'xx', 'yy'}
+
+
+def test_preprocess_drops_coords_that_are_not_plotted_custom_coords():
+    da = data_array(ndim=2)
+    da.coords['xx2'] = 7.5 * da.coords['xx']
+    da.coords['yy2'] = 3.3 * da.coords['yy']
+    out = preprocess(da, coords=['yy2', 'xx'])
```

Comment:
Also test the case where the arg passed as `coords` is a string?
e.g. `preprocess(da, coords='yy2')`

Reply:
Done